array_api_compat/common/_linalg.py

```diff
@@ -174,3 +174,5 @@ def trace(
            'svd', 'cholesky', 'matrix_rank', 'pinv', 'matrix_norm',
            'matrix_transpose', 'svdvals', 'vecdot', 'vector_norm', 'diagonal',
            'trace']
+
+_all_ignore = ['math', 'normalize_axis_tuple', 'get_xp', 'np', 'isdtype']
```

Comment:
No idea why `test_all` wasn't failing before.

Reply:
Yeah, `test_all` is a bit brittle. At a guess, its iterating over `sys.modules` is meant to be robust against renames/new private submodules in `array_api_compat/*`. But then it's not easy to know if something was or was not imported.